tests/cases/fourslash/docCommentTemplateClassDeclProperty01.ts

```diff
@@ -0,0 +1,28 @@
+/// <reference path='fourslash.ts' />
+
+const singleLineOffset = 3;
+const multiLineOffset = 12;
+
+
+////class C {
+////    /** /*0*/  */
+////    foo = (p0) => {
+////        return p0;
+////    };
+////    /*1*/
+////    bar = (p1) => {

```

Comment:
Should you also test the function expression case?

Reply:
Yes!